src/letters/WordDictionary3.js

```diff
@@ -0,0 +1,1072 @@
+let wordDictionary3 = [
```

Comment:
it's up to you, you can keep it this way if its more readable, but if you removed the spaces it would save a lot of space for these giant files

Reply:
I'll pass on this idea a I don't know how to do this other than manually.